app/services/materialized_path_updater_service.rb

```diff
@@ -0,0 +1,60 @@
+# Generates a materialized path for all of the goods nomenclatures under a given chapter
+#
+# We accumulate updates inside of updated_goods_nomenclatures so we can execute one SQL statement for each goods nomenclature that has their path updated without slowing ourselves down with lots of queries.
```

Comment:
this is a long line, should we split it?

Reply:
Oh interesting question. I don't really mind with comments but can do